README.md

```diff
@@ -1,19 +1,2 @@
-## Google Cloud Platform Go Samples
-
-This repository holds the samples used in the Go documentation on [cloud.google.com](https://cloud.google.com).
-
-[![Build Status](https://travis-ci.org/GoogleCloudPlatform/go-docs-samples.svg)](https://travis-ci.org/GoogleCloudPlatform/go-docs-samples)
-
-
-See [golang-samples](https://github.com/GoogleCloudPlatform/golang-samples)
-and our other [Google Cloud Platform github
-repos](https://github.com/GoogleCloudPlatform) for sample applications and
-scaffolding for other frameworks and use cases.
-
-## Contributing changes
-
-* See [CONTRIBUTING.md](CONTRIBUTING.md)
-
-## Licensing
-
-* See [LICENSE](LICENSE)
+Deprecated. Moved here:
+https://github.com/GoogleCloudPlatform/golang-samples
```

Comment:
There's an icon and stuff at go/gcp-github#heading=h.p3rm86v8wss8 that you can use.